controllers/device_controller.go

```diff
@@ -205,14 +188,14 @@ func (r *DeviceReconciler) reconcileCreateDevice(ctx context.Context, d *devicev
 	return r.Status().Update(ctx, d)
 }
 
-func (r *DeviceReconciler) reconcileUpdateDevice(ctx context.Context, d *devicev1alpha1.Device, log logr.Logger) error {
+func (r *DeviceReconciler) reconcileUpdateDevice(ctx context.Context, d *devicev1alpha1.Device) error {
 	// the device has been added to the edge platform, check if each device property are in the desired state
 	newDeviceStatus := d.Status.DeepCopy()
 	// This list is used to hold the names of properties that failed to reconcile
 	var failedPropertyNames []string
 
 	// 1. reconciling the AdminState and OperatingState field of device
-	log.V(3).Info("reconciling the AdminState and OperatingState field of device")
+	klog.V(3).Info("reconciling the AdminState and OperatingState field of device")
```

Comment:
Adding device name into log message may be better.

Reply:
The deviceName has been added.